algorithms/src/sorting/Kokkos_BinSortPublicAPI.hpp

```diff
@@ -60,12 +60,8 @@ class BinSort {
         Kokkos::is_view<SrcViewType>::value,
         Kokkos::View<typename SrcViewType::const_data_type,
                      typename SrcViewType::array_layout,
-                     typename SrcViewType::device_type
-#if not defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC
-                     ,
-                     Kokkos::MemoryTraits<Kokkos::RandomAccess>
-#endif
```

Comment:
Why do we not check for the version here but in other places?

Reply:
Yeah gonna fix it sorry.